src/applications/pensions/config/chapters/02-military-history/generalHistory.js

```diff
@@ -19,7 +21,7 @@ export default {
     type: 'object',
     required: ['serveUnderOtherNames'],
     properties: {
-      serveUnderOtherNames: yesNoSchema,
+      serveUnderOtherNames,
```

Comment:
This seems like an improvement to me - interested to hear if @ToddWebDev sees any downsides

Reply:
The only downside of this pattern I can see is that it abstracts the schema from vets-website a bit, but I don't think it's a deal breaker as it is more inline with the vets-json-schema.